test/libsolidity/ASTPropertyTests/nested_properties.sol

```diff
@@ -0,0 +1,17 @@
+contract C {
+    function f() public pure {
+        /// TestCase1: condition.operator
+        for(uint i = 0; i < 42; ++i) {
+        }
+        /// TestCase2: initializationExpression.initialValue.value
+        for(uint i = 1; i < 42; i = i * 2) {
+         }
+         /// TestCase3: loopExpression.expression.subExpression.name
+         for(uint i = 0; i < 42; i++) {
+         }
```

Comment:
Is this not indented properly to test `trim()`, or are you trying to summon Kamil?

Reply:
Are you referring to the closing brackets being in a new line?

Reply:
Oh, right. I could not see that until now! :smile: